### PR TITLE
Updating README

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

**Comment by gvn** [comment]
@simonwex R?

File: README.md [hunk 1]
```diff
@@ -1,1527 +1,22 @@
...
-    + `number`
-    + `boolean`
-    + `null`
-    + `undefined`
+## Rules
 
-    ```javascript
-    var foo = 1;
-    var bar = foo;
+Refer to the [RC files](https://github.com/MozillaFoundation/javascript-style-guide/tree/master/linters) for specifics.
```
Comment (gvn) [inline README.md]: I figure at least while things are in flux this will keep things from getting out of sync...

**Comment by gvn** [comment]
Resolves #10

**Comment by simonwex** [comment]
I hate loosing the human-readable explanation, though I understand we're out of sync. Leave this one with me for the week, I'll see if I can make headway.

**Comment by gvn** [comment]
So...been a few months. Any objection to landing this? @simonwex

**Comment by simonwex** [comment]
Go for it.

**Pull request merged**